libres/testjenkins.sh

```diff
@@ -109,7 +108,6 @@ clone_repos () {
 	echo "Cloning into $LIBRES_ROOT"
 	git clone . $LIBRES_ROOT
 	mkdir -p $LIBRES_BUILD
-	ln -s /project/res-testdata/ErtTestData $LIBRES_ROOT/../test-data/Equinor
```

Comment:
Why are we removing instead of fixing this?

Reply:
I am doing some tests. Adding equinor-data was done three places. One in the cloning, one at compile time, and one right before testing. Only one of those are needed. I am suspecting that trying to do this redundantly might cause some issues. Will poke you when I find out. I suspect that the compile time is never used as LD_LIBRARY_PATH is not set to the newly built libres, and that the second linking causes the run_pytest_equinor is terminated when the linking fails.

Reply:
This worked: (https://ci.equinor.com/scout/job/ert_PR/1319/console)
```
../../libres/tests/res/enkf/test_ecl_config.py ...                       [  5%]
../../libres/tests/res/enkf/test_enkf_fs.py ....                         [ 12%]
../../libres/tests/res/enkf/test_enkf_library.py ..                      [ 15%]
../../libres/tests/res/enkf/test_enkf_load_results_manually.py ..        [ 19%]
../../libres/tests/res/enkf/test_enkf_obs.py ..s....                     [ 31%]
../../libres/tests/res/enkf/test_ert_context.py ...                      [ 36%]
../../libres/tests/res/enkf/test_field_export.py s...                    [ 43%]
../../libres/tests/res/enkf/test_labscale.py ss                          [ 47%]
../../libres/tests/res/enkf/test_summary_key_set.py ....                 [ 54%]
../../libres/tests/res/enkf/test_update.py s                             [ 56%]
../../libres/tests/res/enkf/data/test_gen_data.py .                      [ 57%]
../../libres/tests/res/enkf/data/test_gen_data_config.py ..              [ 61%]
../../libres/tests/res/enkf/data/test_gen_kw_config_equinor.py .         [ 63%]
../../libres/tests/res/enkf/export/test_arg_loader.py .                  [ 64%]
../../libres/tests/res/enkf/plot/test_plot_data.py ...                   [ 70%]
../../libres/tests/res/fm/test_ecl_run.py ........                       [ 84%]
../../libres/tests/res/fm/test_ecl_run_new.py .........                  [100%]
```
but I will test some more